jericho/app/src/database.py

```diff
@@ -1,8 +1,6 @@
-from typing import Optional
-
+from src.config import pg_url
 from sqlmodel import Field, SQLModel, UniqueConstraint, create_engine
-
-from sql.config import pg_url
+from typing import Optional
 
 
 class Book(SQLModel, table=True):
```

Comment:
this is less than ideal pydantic style but should get resolved soon

re: https://sqlmodel.tiangolo.com/tutorial/fastapi/